<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>softwareSourceDlg</class>
 <widget class="QDialog" name="softwareSourceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#softwareSourceDlg
{
background-color: rgb(211, 211, 211);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/widget/img/widget/back.png) 6 352 18 0;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CWidget" name="backWdg" native="true">
     <property name="styleSheet">
      <string notr="true">#backWdg
{
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_9" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(82, 204, 144);
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="dddd">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 16px &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>软件源</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClose">
           <property name="minimumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{	
	border-image: url(:/Login/img/Login/close_green_Normal.png);
}

QPushButton:hover{		
	border-image: url(:/Login/img/Login/close_green_Click.png);
}

QPushButton:pressed{
	border-image: url(:/Login/img/Login/close_green_Normal.png);
}

QPussButton:disable{
	border-image: url(:/Login/img/Login/close_green_Normal.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>36</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="widget_6" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="widget_7" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="horizontalSpacing">
             <number>5</number>
            </property>
            <property name="verticalSpacing">
             <number>10</number>
            </property>
            <item row="2" column="1">
             <widget class="QCheckBox" name="ckb_compatible">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">#ckb_compatible{
font: 12px &quot;微软雅黑&quot;;
color: rgb(68, 68, 68);
}</string>
              </property>
              <property name="text">
               <string>兼容模式(适用于演示office、QQ等软件)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QWidget" name="widget_8" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="horizontalSpacing">
                <number>20</number>
               </property>
               <property name="verticalSpacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QComboBox" name="mCmbWindowList">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">
QComboBox {
font: 14px &quot;微软雅黑&quot;;
    border: 1px solid rgb(221, 221, 221);
    border-radius: 3px;
    padding: 1px 1px 1px 5px;
    background-color: rgb(255, 255, 255);
	color: rgb(68, 68, 68);
}

QComboBox::down-arrow {
	image: url(:/ctrl/img/ctrl/media_setting_down.png);

}
QComboBox::drop-down{
    border:0px;
}

QComboBox::up-arrow {
	image: url(:/ctrl/img/ctrl/media_setting_up.png);
}

QComboBox QAbstractItemView{
    outline: 0px;
    border-radius:4px;
	color: rgb(68, 68, 68);
	selection-background-color: rgb(82, 204, 144);
	selection-radius:4px;
	background-color: rgb(255, 255, 255);
	border-left:15px;
 border: 1px solid rgb(221, 221, 221);
}

/*滚动条样式 开始*/
QScrollBar:vertical
{
border-radius:4px;
width:6px;
background:rgba(255,255,255, 255);
margin:0px,0px,0px,0px;

}

QScrollBar::handle:vertical
{
background:rgba(210,210,210, 255);
border-radius:3px;
min-height:20;
border: 1px solid #D2D2D2;
}

QComboBox:!enabled {
    background-color: rgb(232, 232, 232);
    border: 1px solid rgb(210, 210, 210);
	color: rgb(153, 153, 153);
}
</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="btnFlush">
                 <property name="minimumSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
border-image: url(:/button/img/button/Refresh_Normal.png);
}

QPushButton:hover{
border-image: url(:/button/img/button/Refresh_Click.png);
}

QPushButton:pressed{
border-image: url(:/button/img/button/Refresh_Normal.png);
}
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="softwareList_Label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#softwareList_Label{
font: 14px &quot;微软雅黑&quot;;
color: rgb(68, 68, 68);
}</string>
              </property>
              <property name="text">
               <string>请选择软件源</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="saveBtn">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{	
	color: rgb(255, 255, 255);
	border:0px;
	border-radius:4px;
	background-color: rgb(82, 204, 144);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	border:0px;
	border-radius:4px;
	background-color: rgb(52, 184, 119);
}

QPushButton:disable{
	color: rgb(255, 255, 255);
	border:0px;
	border-radius:4px;	
	background-color: rgb(99, 224, 162);
}

</string>
           </property>
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/widget/img/widget/back.png) 10 0 18 352;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>10</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/widget/img/widget/back.png) 0 0 194 0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_7">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/widget/img/widget/back.png) 182 0 0 0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CWidget</class>
   <extends>QWidget</extends>
   <header>cwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../VLive/VLive.qrc"/>
 </resources>
 <connections/>
</ui>
